marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/mapping/entity-services/function-metadata.xsl

```diff
@@ -97,7 +97,7 @@
               <xsl:value-of select="@name"/>
             </xsl:attribute>
             <xsl:attribute name="select">
-              <xsl:value-of select="concat('($',@name,',null-node{})[1]')"/>
+                <xsl:value-of select="concat('if (empty($', @name ,')) then null-node{} else $', @name)"/>
```

Comment:
@ryanjdew
This seems to work fine. I didn't have to add xsl:choose element. Let me know if you see any issues with this.